apps/vc-api/docs/example-exchange-definitions/presentation-exchange.md

```diff
@@ -0,0 +1,55 @@
+Sample Exchange Definition to make Verifiable Presentation with Permanent Resident Card.
```

Comment:
Can you align the wording between this file and the `one-presentation-one-self-signed-exchange`?
Here we have the presentation described as "make Verifiable Presentation with Permanent Resident Card." and in the other file is described as "presentation (VC issued by authority through VP)"

Reply:
I will make it general as done in other files like `Sample Exchange Definition for presentation and self-signed credential`.